Formula/tezos-node-edo2net.rb

```diff
@@ -4,7 +4,7 @@
 
 class TezosNodeEdo2net < Formula
   url "file:///dev/null"
-  version "v8.2-3"
+  version "v9.0-rc1-1"
```

Comment:
v9.0-rc1 now supports `--network edo2net`, so I propose to remove ugly workarounds with `cat`ing the required node config in this PR

Reply:
:ok_hand: good point, done